src/components/forms/checkbox/RuiCheckbox.vue

```diff
@@ -157,6 +157,16 @@ watch(internalModelValue, (val) => {
 <style lang="scss" module>
 @use '@/styles/colors.scss' as c;
 
+:global(.dark) {
+  .wrapper {
+    .checkbox {
+      &:before {
+        @apply bg-white;
+      }
+    }
+  }
+}
```

Comment:
ok so quick question that I had in mind before, instead of global(.dark) is there any other way could use this style in `.wrapper .checkbox &:before { }`

could we apply `dark:bg-white` there to reduce the extra lines?